src/modules/75-cf/pages/feature-flags/components/__tests__/FlagTableFilters.test.tsx

```diff
@@ -54,6 +56,8 @@ describe('FlagTableFilters', () => {
     })
   })
 
+  beforeEach(() => (flagEnabled = true))
```

Comment:
I'd reckon Sonar will complain about this
```suggestion
  beforeEach(() => {
    flagEnabled = true
  })
```

Reply:
I don't care for fixing this as it's acceptable on one line, Sonar can complain like it does about everything else